app/assets/stylesheets/sass/main.scss

```diff
@@ -0,0 +1,3 @@
+@import 'base/_typography';
+@import 'pages/employees';
+@import 'pages/login';
```

Comment:
I like the approach you are using here, I'd remove the `sass` directory and set all these files in the `stylesheets` directory:
from:
```
 - app/assets/stylesheets/sass/main.scss
 - app/assets/stylesheets/sass/base/_typography.scss
 - app/assets/stylesheets/sass/pages/employees.scss
 - app/assets/stylesheets/sass/pages/login.scss
```
to:
```
 - app/assets/stylesheets/main.scss
 - app/assets/stylesheets/base/_typography.scss
 - app/assets/stylesheets/pages/employees.scss
 - app/assets/stylesheets/pages/login.scss
```

Reply:
Is a suggestion, what do you think?

Reply:
I agree, I'll change this, we use the approach you posted on slack, it's very clear